snuba/datasets/configuration/generic_metrics/entity_generic_metrics_distributions_friendlier.yaml

```diff
@@ -0,0 +1,29 @@
+schema: v1
+kind: entity
+name: generic-metrics-distributions
+spec:
+  readable_storage: distributions_storage
+  writable_storage: distributions_bucket_storage
+  query_processors:
+    -
+      transform_tag_types:
+    -
+      handle_mapped_granularities:
+        accepted_granularities:
+          -
+            - 60
+            - 1
+          -
+            - 3600
+            - 2
+          -
+            - 86400
+            - 3
```

Comment:
this looks a little weird

Reply:
yeah, it probably makes more sense as a mapping e.g.

```
accepted_granularities:
  60: 1
  3600: 2
  86400: 3
```